storey/aggregation_utils.py

```diff
@@ -87,3 +87,21 @@ def get_dependant_aggregates(aggregate):
         if aggr_bits & raw_aggr == raw_aggr:
             aggrs.append(_all_aggregates_to_name[raw_aggr])
     return aggrs
+
+
+def get_all_raw_aggregates_with_hidden(aggregates):
```

Comment:
A bit confusing... It looks like this return a map from aggr to whether or not it's dependant (not raw? aka "hidden"?).

Reply:
it does returns a dict of raw aggregates and whether or not they are hidden or not.
If you want to change the function name and have ideas bekeif

Reply:
dependant -> implied?